WordPress/src/main/java/org/wordpress/android/ui/posts/reactnative/ReactNativeUrlHandler.kt

```diff
@@ -0,0 +1,48 @@
+package org.wordpress.android.ui.posts.reactnative
+
+import org.wordpress.android.util.AppLog
+import org.wordpress.android.util.AppLog.T.EDITOR
+import java.lang.IndexOutOfBoundsException
+
+private const val WPCOM_ENDPOINT = "https://public-api.wordpress.com"
+
+internal fun parseUrlAndParamsForWPCom(pathWithParams: String, wpComSiteId: Long): Pair<String, Map<String, String>>? =
```

Comment:
I'd change this file to an injected Utils class so we don't have to retest its functionality in all the places where it's called.

Reply:
I wanted to avoid creating one of those util classes that doesn't have any state or any dependencies because those types of classes are one of the few cases where imho testing considerations lead to worse code design. Of course, I also didn't realize that we weren't using powermock or mockk in this project 😄 , and I certainly understand there are also good reasons for avoiding those libraries. 

Updated to be an injectable `ReactNativeUrlUtil` class. 👍